SparseGrids/tasgridUnitTests.cpp

```diff
@@ -786,4 +793,57 @@ std::vector<std::function<void(void)>> GridUnitTester::getRuntimeErrorCalls() co
     };
 }
 
+bool GridUnitTester::testLAPACKInterface() {
+    bool all_matched = true;
+
+    #ifdef Tasmanian_ENABLE_BLAS
```

Comment:
Adding the pre-processor here is good. Also make sure that the BLAS header is not included unless BLAS is declared.

Reply:
Gotcha. This is handled with ``import tsgTPLWrappers.hpp`` now.